mmdet3d/datasets/pipelines/transforms_3d.py

```diff
@@ -116,6 +116,71 @@ def __repr__(self):
         return repr_str
 
 
+@PIPELINES.register_module()
+class RandomFlipMono3D(RandomFlip3D):
```

Comment:
Can we simply enhance the original RandomFlip3D module?

Reply:
Enhanced and merged.